DEV-4/NondecreasingSequence/Entrypoint.cs

```diff
@@ -17,7 +17,7 @@ static void Main(string[] args)
                 {
                     Console.WriteLine(ENTER_SEQUENCE);
                     InputSequence inputsequence = new InputSequence();
-                    int[] SequenceArray = inputsequence.Input();
+                    int[] SequenceArray = inputsequence.Input(args);
```

Comment:
Bad name, I am not sure that it works according to the task, you only change the input type but not provide two possible input types